zcash_keys/src/address.rs

```diff
@@ -119,7 +134,10 @@ impl UnifiedAddress {
                 #[cfg(feature = "sapling")]
                 sapling,
                 transparent,
-                unknown: vec![],
+                unknown_data: vec![],
+                expiry_height,
+                expiry_time,
+                unknown_metadata: vec![],
             })
```

Comment:
Change this to allow creating a UA with only a transparent receiver plus expiration metadata, consistent with the suggested doc change.